### ISSUE TITLE
New environment variables

### ISSUE THREAD

**Issue by Nico83500** [nextcloud/docker#1547]
Hi,
I would like to know if there are some env var to configure : 
- Choose to install or not recommended apps ? Because I use admin and password env var and I don't know if recommended apps are installed
- overwrite.cli.url ? I don't know if Nextcould uses it but the value is  'overwrite.cli.url' => 'http://localhost', in config.php
- Installation or skip warning about imagemagick : I can see there is already an issue about it but no news ?
- Add secrets support for SMTP env var ? I believe I already seen an issue about it too
Thank you !

**Comment by J0WI** [comment]
Please have a look at our [README](https://github.com/nextcloud/docker#auto-configuration-via-environment-variables). There is no env var for imagemagick or recommended apps.

**Comment by MarcoLeongDev** [comment]
Agree, it is clear in the doc shows that `overwrite.cli.url` isn't there for environment variable.

However, for the use case of SSL + mobile Apps, `overwrite.cli.url` is essential to be part of the environment variable so all the setup can be completed in the compose file, instead of modifying `config/config.php` after the deployment.

Is this feature request worthy? Otherwise, what's the reason?
